src/ix_stresser/pdfbulknode.clj

```diff
@@ -0,0 +1,87 @@
+(ns ix-stresser.pdfbulknode
+  (:require [clj-http.client :as client]
+            [clojure.core.async :as async :refer [<! <!! go go-loop timeout]]
+            [clojure.data.json :as json]))
+
+(def base-files-path "./doc/")
+(def dev-url "http://localhost:3000")
+(def stag-url "http://pdfnode-staging.cloudapp.net")
+(def prod-url "http://pdfnode.cloudapp.net")
+(def existing-files ["invoice_small.json", "invoice_medium.json", "invoice_big.json"])
+
+(def invoice-on-bulk 1) ;; 1 will mean a single bulk having 3 invoices, small, medium and big
+;; 2 will mean 6 invoices, 2 small, 2 medium , 2 big... and so on...
+
+(def stressing-times 1)
+(def requesting-times 1)
+
+(defn url-for [url, endpoint]
+  (str url endpoint))
+
+(defn read-file [filename]
+  (json/read-str (slurp (str base-files-path filename)) :key-fn keyword))
+
+(defn read-all-files []
+  (reduce (fn [hash filename]
+            (assoc hash (keyword filename) (read-file filename)))
+          {}
+          existing-files))
+
+(defn bulk-contents-data []
+  (reduce (fn [array filename]
```

Comment:
yes, it will be another data transformation but I'm not seeing how... :P
I put the `->>` on this `bulk-contents-data` method because of the three steps of the `reduce fn`... how can I put the reduce too inside of it?